README.md

```diff
@@ -10,51 +10,88 @@ __The project is in its *very* early stages, there is nothing to see here yet.__
 
 Tobira consists of two parts:
 
+- `frontend` contains the web interface written in TypeScript/React.
 - `backend` contains the video portal backend written in Rust which takes care of data persistence and communication to Opencast.
-- `frontend` contains the web interface written in Typescript/React.
 
 
 ## How to build Tobira
 
-### Install Development Dependencies
+### 1. Development Tools
 
-You will need `rustc` and `cargo` to build the Tobira backend.
-To get them, use [`rustup`](https://rustup.rs) or your package manager.
+Make sure you have the following tools installed:
 
-You will need NPM to build the frontend.
+- backend
+    - `rustc`
+    - `cargo`
+- frontend
+    - `node`
+    - `npm`
+- optional
+    - [`floof`](https://github.com/LukasKalbertodt/floof)
 
 
-### Local Database
+### 2. Database
 
-To run Tobira, you need a PostgreSQL database.
-You can easily launch a database using docker-compose:
+Tobira needs a PostgreSQL database.
+The `scripts` directory contains a container compose file to easily spin one up.
 
 ```sh
-cd scripts
+cd scripts/
+# using docker
 docker-compose up -d
+# using podman
+podman-compose up -d
 ```
 
-To shut down the container again:
```

Comment:
Why remove this section? :thinking:

Reply:
Several reasons:

1. Same as above: I't unnecessary additional information people can easily find out and it's not our task to explain container orchestration here. Thus, I keep that to a minimum.
2. You don't need this for getting started at all. Removing this will not hinder people in getting Tobira built and run
3. This is a step-by-step instruction. Shutting things down is not something you _should_ do here